backend/utils/cloudinary.py

```
@@ -0,0 +1,26 @@
+import cloudinary
+import cloudinary.uploader
+from fastapi import HTTPException
+
+cloudinary.config(
+  cloud_name="dws8lpmua",
+  api_key="595136529363616",
+  api_secret="VwJl7HXcox_1U9qeeN0fVmoL_VY",
+  secure=True
+)
+
+def upload_to_cloudinary(file, folder_name):
```

Ruff (0.8.2)

Missing return type annotation for public function `upload_to_cloudinary`

(ANN201)